src/dhtnode/storage/DumpManager.d

```diff
@@ -586,6 +594,8 @@ version ( UnitTest )
     import dhtnode.node.DhtHashRange;
     import ocean.io.serialize.SimpleStreamSerializer : EofException;
 
+    const RecordSizeLimit = 1024;
```

Comment:
Should we still hard code this number here?

Reply:
Yes I explicitly added it for the unittest.

Reply:
I'll add a comment to make it clear.